Lab 2.1.py

```diff
@@ -0,0 +1,17 @@
+def search(products, product):
+    try: return products.index(product)
+    except ValueError: return None
+
+product_names = [
+    "USB Flash Drive",
+    "Speaker",
+    "Webcam",
+    "Webcam",
+    "Microphone",
+    "Projector",
+    "Laptop",
+    "Microphone",
+    "Drone",
+    "USB Flash Drive"
+]
+print(search(product_names, 'Drone'))

```

Comment:
Данное решение выполняет поставленные функции, но не соответствует заданию в рамках курса.
На курсе представленное задание выглядит следующим образом:
```suggestion
def find_item_index ... # TODO Написать функцию поиска индекса 
items_list = ['яблоко', 'банан', 'апельсин', 'груша', 'киви', 'банан']
for find_item in ['банан', 'груша', 'персик']:
    index_item = find_item_index(items_list, find_item)  # TODO Вызовите функцию, что получить индекс товара
    if index_item is not None:
        print(f"Первое вхождение товара '{find_item}' имеет индекс {index_item}.")
    else:
        print(f"Товар '{find_item}' не найден в списке.")

```